cmd/svcat/svcat_test.go

```diff
@@ -85,6 +85,10 @@ func TestCommandOutput(t *testing.T) {
 		{name: "get plan by name", cmd: "get plan default", golden: "output/get-plan.txt"},
 		{name: "get plan by uuid", cmd: "get plan --uuid 86064792-7ea2-467b-af93-ac9694d96d52", golden: "output/get-plan.txt"},
 		{name: "get plan by class/plan name combo", cmd: "get plan user-provided-service/default", golden: "output/get-plan.txt"},
+		{name: "get plan by class name", cmd: "get plan --class user-provided-service", golden: "output/get-plans.txt"},
```

Comment:
Due to our limited testdata (there is only a single plan in the ups broker), this test doesn't actually catch when we didn't filter the plans by class appropriately. I recommend adding another class to the recorded json responses in the testdata directory, renaming the golden file for this test case to `output/get-plans-by-class.txt` and then rerunning the tests `go test ./... -update` with the update flag to update the golden files. 

Then you should see different output between the get-plans.txt and get-plans-by-class-name.txt demonstrating that the filtering works.